db/migrate/20170530180114_add_manufacturer_and_fru_to_guest_devices.rb

```diff
@@ -0,0 +1,8 @@
+class AddManufacturerAndFruToGuestDevices < ActiveRecord::Migration[5.0]
+  def change
+    add_column :guest_devices, :manufacturer, :string
+    add_column :guest_devices, :fru, :string
+    add_column :guest_devices, :guest_device_id, :bigint
+    add_index :guest_devices, :guest_device_id, :name => "index_guest_devices_on_guest_device_id"
```

Comment:
@Fryguy I believe that I had to explicitly define the guest_device_id column for the "guest_device to guest_device" relationship to work. There were errors in the logs about it not existing.